docs/concepts/standard-library/overview.md

```diff
@@ -0,0 +1,75 @@
+---
+id: overview
+---
+
+# Overview
+
+Gno comes with a set of standard libraries which are included to ease development and provide extended functionality to the language. These include:
+- standard libraries as we know them in classic Golang, i.e. `encoding/binary`, `strings`, `testing`, etc.
+- a special `std` package, which contains types, interfaces, and APIs created to handle blockchain-related functionality.
+
+
+Standard libraries differ from on-chain packages in terms of their import path structure.
+Unlike on-chain [packages](../packages.md), standard libraries do not incorporate a domain-like format at the beginning
+of their import path. For example:
+- `import "encoding/binary"` refers to a standard library
+- `import "gno.land/p/demo/avl"` refers to an on-chain package.
```

Comment:
```suggestion
- `import "gno.land/p/demo/avl"` refers to an on-chain package.
```

Maybe see what others think about calling this a userland package for uniformity whenever we speak of these types of 3rd party packages.

Reply:
Let's bring this up on a call. Resolving to merge.